src/methods/seek.js

```diff
@@ -29,6 +29,17 @@ const seek = ({ media, direction, seekSec, cacheRequired = false }) => {
   }
 }
 
+/**
+ * @param {HTMLVideoElement} media - Video element to be handled
+ * @param {number} numericKey - Number key pressed. Must be between 0 and 9
```

Comment:
```suggestion
 * @param {string} numericKey - Number key pressed. Must be between 0 and 9
```

Values passed are string actually.

Reply:
How shameful!